Roadmap.md

```diff
@@ -12,7 +12,7 @@ This document outlines the roadmap of the Mixed Reality Toolkit.
 | --- | --- | --- | --- |
 |MRTK V2 RC 1 | Add HoloLens 2 support. API/contract complete. No more breaking changes after this | March 2019 |  |
 |MRTK V2 RC 2| UX update and polish to better support HoloLens 2 interactions. | May 2019 |  |
-|MRTK V2 Release | First release of MRTK V2 that supports HoloLens 1, HoloLens 2, WMR, OpenVR | July 2018 | |
+|MRTK V2 Release | First release of MRTK V2 that supports HoloLens 1, HoloLens 2, WMR, OpenVR | July 2019 | |
```

Comment:
Does this change need to happen in this PR?

Reply:
i can remove it, but it already exists in mrtk_development in the microsoft repo. Do we think this would make for a harder merge?

Reply:
I've got the latest from the development branch, so this is a new change.

Reply:
@keveleigh is going to merge mrtk_release into mrtk_development. That should account for these commits.

Reply:
https://github.com/Microsoft/MixedRealityToolkit-Unity/pull/3619